lib/postgresql/native_postgres.nit

```diff
@@ -83,27 +85,27 @@ end
 extern class NativePostgres `{PGconn *`}
 
   # Connect to a new database using the conninfo string as a parameter
-  new connectdb(conninfo: String) import String.to_cstring `{
+  new connectdb(conninfo: Text) import String.to_cstring `{
     PGconn * self = NULL;
     self = PQconnectdb(String_to_cstring(conninfo));
```

Comment:
Since the type of `conninfo` changed, you must also change this callback and related import to `Text.to_cstring`.